server/main.go

```diff
@@ -164,7 +172,17 @@ func main() {
 			lockedAccounts[i] = common.HexToAddress(l).Hex()
 		}
 
-		backendInstance = backend.NewBackend(mongoUrl, rpcUrl, dbName, lockedAccounts)
+		data, err := ioutil.ReadFile(configFile)
```

Comment:
This whole block should only be attempted if the value is set, leaving `nodes` nil otherwise (or maybe an empty map if necessary).
```go
var nodes map[common.Address]models.Node
if configFile != "" {
  ...
}
```